docs/auth/microsoft/azure-easyauth.md

```diff
@@ -0,0 +1,120 @@
+---
+id: azure-easy-auth
+title: Azure EasyAuth Provider
+sidebar_label: Azure EasyAuth
+description: Adding Azure's EasyAuth Proxy as an authentication provider in Backstage
+---
+
+## Backstage Changes
+
+Add the following into your `app-config.yaml` or `app-config.production.yaml` file
+
+Add a `providerFactories` entry to the router in
+`packages/backend/src/plugins/auth.ts`.
+
+```ts
+import { providers } from '@backstage/plugin-auth-backend';
+
+export default async function createPlugin(
+  env: PluginEnvironment,
+): Promise<Router> {
+  const authProviderFactories = {
+    'azure-easyAuth': providers.easyAuth.create({
```

Comment:
having case sensitive things here, which in turn get transferred to routes is a bit of a codesmell for me.

thoughts on either `azure-easy-auth` or `azure-easyauth`?